debian/changelog

```diff
@@ -1,3 +1,7 @@
+  [Marco Pagani, Christopher Brooks]
+  * Refactored Chiou and Youngs (2014) GMM to be adjustable using methodology
+  of the Boore et al (2022) paper
```

Comment:
The indentation of the second line is wrong and this may cause issues to a `sed` script used by Matteo to process the changelog.

Reply:
Just fixed now thanks